Upgrade EUI to v101.4.0

mgadewoll opened the pull request:
`101.3.0` ⏩ `101.4.0`

[Questions? Please see our Kibana upgrade FAQ.](https://github.com/elastic/eui/blob/main/wiki/eui-team-processes/upgrading-kibana.md#faq-for-kibana-teams)

## Changes

This PR only updates tests and snapshots related to changes on EUI side:
- updated `aria-current="true"` to `aria-current="page"`
- updated icon usage `userAvatar` to `user`

## Package updates

### `@elastic/eui`

#### [`v101.4.0`](https://github.com/elastic/eui/releases/v101.4.0)

- Spread `labelProps` to the `label` element in `EuiCheckableCard` ([#8586](https://github.com/elastic/eui/pull/8586))
- Add `controls`, `flask`, `comment`, and `readOnly` glyphs to `EuiIcon` ([#8580](https://github.com/elastic/eui/pull/8580))
- Refactored `EuiExpression`, `EuiFacetGroup`, `EuiFacetButton`, `EuiFilterGroup`, `EuiHeader`, `EuiImage` and `EuiListGroup` to memoize their internal Emotion styles ([#8565](https://github.com/elastic/eui/pull/8565))
- Updated global `border.radius.medium` token value for default `Borealis` theme to `4px` ([#8563](https://github.com/elastic/eui/pull/8563))
- Updated `EuiProvider` to build themes including `highContrastMode` ([#8558](https://github.com/elastic/eui/pull/8558))

**Accessibility**

- Removed the `aria-label` attribute from the `ul` element in `EuiPagination` to avoid duplicate screen reader output ([#8597](https://github.com/elastic/eui/pull/8597))
- Set a more specific `aria-current="page"` on list items in `EuiPagination` ([#8597](https://github.com/elastic/eui/pull/8597))
- Added `aria-modal` to `EuiFlyout` with `type="overlay"` ([#8591](https://github.com/elastic/eui/pull/8591))

**Dependency updates**

- Updated `@elastic/prismjs-esql` to v1.1.0 ([#8587](https://github.com/elastic/eui/pull/8587))

### `@elastic/eui-theme-borealis@0.2.0`

- Updated component tokens to use `computed` values to ensure correct inheritance from theme overrides ([#8558](https://github.com/elastic/eui/pull/8558))
- Added `overrides.HCM` to `euiThemeBorealis` to support theme internal overrides ([#8558](https://github.com/elastic/eui/pull/8558))
- Updated `border.radius.medium` token value to `4px` ([#8563](https://github.com/elastic/eui/pull/8563))

### `@elastic/eui-theme-common@0.2.0`

- Added support for theme `overrides` as optional part of `EuiThemeShape` ([#8558](https://github.com/elastic/eui/pull/8558))
- Updated `getComputed` to support high contrast mode overrides defined on `overrides.HCM` ([#8558](https://github.com/elastic/eui/pull/8558))

<!--ONMERGE {"backportTargets":["9.0"]} ONMERGE-->

mgadewoll committed:
chore: update eui packages

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294792)
* Commit: 7f7ea97b6c8f5bf82678f0929a583edca73e71c4

### Failed CI Steps
* [Jest Tests #2](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260)
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845)
* [Jest Tests #8](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4c31-9f18-b67e68b4f1da)
* [Jest Tests #11](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d)
* [Jest Tests #12](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06)
* [Jest Tests #18](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-4864-92c7-0eaead001012)
* [Jest Tests #18](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965ceb-8d3a-46b9-9677-052c10a1cafa)
* [Jest Tests #19](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1)
* [FTR Configs #59](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd27-420e-89cb-608df6fc3c58)
* [FTR Configs #110](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd49-463d-8b36-646b7d952f70)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e404-4f9f-add4-1e24ddbc5357) Jest Tests #<span></span>12 / AlertsGrouping resets all levels pagination when global query updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e402-409d-a15d-b22de5ce0289) Jest Tests #<span></span>12 / AlertsGrouping resets all levels pagination when selected group changes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e403-4d1e-986c-9f2c4c30bcf5) Jest Tests #<span></span>12 / AlertsGrouping resets innermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e404-45cf-a546-c4cda870a24e) Jest Tests #<span></span>12 / AlertsGrouping resets only most inner group pagination when its parent groups open/close
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e403-4de4-a5eb-3bc1786a3502) Jest Tests #<span></span>12 / AlertsGrouping resets outermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d25-43b0-945d-3b709a57828e) Jest Tests #<span></span>19 / Authentication Host Table Component rendering it renders the host authentication table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d22-43b3-b7b8-3a1928f88663) Jest Tests #<span></span>19 / Authentication User Table Component rendering it renders the user authentication table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e405-47ac-9abc-d9bb77ba9cc2) Jest Tests #<span></span>12 / DatatableComponent pagination doesnt change page position when changing the data to a bigger set
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e405-4b09-98f0-350d3965668c) Jest Tests #<span></span>12 / DatatableComponent pagination enables pagination
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-456c-be17-ab427fe2df06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-456c-be17-ab427fe2df06/artifacts/01965cec-e404-4877-808c-86c5d51202b1) Jest Tests #<span></span>12 / DatatableComponent pagination resets page position if rows change so page will be empty
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd27-420e-89cb-608df6fc3c58) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd27-420e-89cb-608df6fc3c58/artifacts/01965cea-c7e6-4cb6-9c33-a3425531bdbf) FTR Configs #<span></span>59 / discover/group2/data_grid2 discover data grid in-table search should show highlights for in-table search
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-057a-4d0d-8f86-c0dbf2b70ee3) Jest Tests #<span></span>2 / EntitiesList should reset the page when sort order changes 
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee2-4e24-a48a-9a2a14233d6b) Jest Tests #<span></span>11 / EQL Tab rendering pagination should load notes for correct page size
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee0-4640-a8b4-c5e47b12f674) Jest Tests #<span></span>11 / EQL Tab rendering pagination should load notes for current page only
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddb-4d5b-8d02-ca2a91d3186d) Jest Tests #<span></span>19 / ExceptionItemCardComments should render comments panel closed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8dde-42ed-bff0-b5160488f9ff) Jest Tests #<span></span>19 / ExceptionItemCardComments should render comments panel opened when accordion is clicked
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-484a-8ee9-49ca0f644845/artifacts/01965ceb-20e8-4a58-81c5-fc54c4e27f7c) Jest Tests #<span></span>7 / GroupedAlertsTable resets all levels pagination when global query updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-484a-8ee9-49ca0f644845/artifacts/01965ceb-20e7-48d3-bd67-e1895f71858c) Jest Tests #<span></span>7 / GroupedAlertsTable resets all levels pagination when selected group changes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-484a-8ee9-49ca0f644845/artifacts/01965ceb-20e6-4987-8603-8f808aa658b4) Jest Tests #<span></span>7 / GroupedAlertsTable resets innermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-484a-8ee9-49ca0f644845/artifacts/01965ceb-20e7-4800-a756-641ba3bb4c88) Jest Tests #<span></span>7 / GroupedAlertsTable resets only most inner group pagination when its parent groups open/close
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-484a-8ee9-49ca0f644845) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-484a-8ee9-49ca0f644845/artifacts/01965ceb-20e6-4409-9979-c8a47a7baa4f) Jest Tests #<span></span>7 / GroupedAlertsTable resets outermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-4864-92c7-0eaead001012) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-4864-92c7-0eaead001012/artifacts/01965ceb-7941-4a55-9201-6b5ea91abe4d) Jest Tests #<span></span>18 / Inspector Data View component should render single table without selector
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965ceb-8d3a-46b9-9677-052c10a1cafa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965ceb-8d3a-46b9-9677-052c10a1cafa/artifacts/01965cf3-295c-4527-b69b-1a6f0fdad2e2) Jest Tests #<span></span>18 / Inspector Data View component should render single table without selector
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd02-4864-92c7-0eaead001012) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd02-4864-92c7-0eaead001012/artifacts/01965ceb-7941-4750-ac66-d17bcc2ea774) Jest Tests #<span></span>18 / Inspector Data View component should support multiple datatables
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965ceb-8d3a-46b9-9677-052c10a1cafa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965ceb-8d3a-46b9-9677-052c10a1cafa/artifacts/01965cf3-295c-49ff-8532-37c1f9644901) Jest Tests #<span></span>18 / Inspector Data View component should support multiple datatables
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d23-4922-8422-166627ae522c) Jest Tests #<span></span>19 / NetworkHttp Table Component rendering it renders the default NetworkHttp table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d22-475e-954f-b9b73dfe7007) Jest Tests #<span></span>19 / NetworkTopCountries Table Component rendering it renders the default NetworkTopCountries table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddb-4ae1-b410-76bb2d01dd0a) Jest Tests #<span></span>19 / NetworkTopCountries Table Component rendering it renders the IP Details NetworkTopCountries table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d24-48d7-8c50-64731177e0bf) Jest Tests #<span></span>19 / NetworkTopNFlow Table Component rendering it renders the default NetworkTopNFlow table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d24-4007-9b8c-10f9733c87bd) Jest Tests #<span></span>19 / NetworkTopNFlow Table Component rendering it renders the default NetworkTopNFlow table on the IP Details page
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddc-4c4c-8785-5cf7101529ed) Jest Tests #<span></span>19 / NetworkTopNFlow Table Component rendering it renders the default NetworkTopNFlow table on the Network page
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8dde-47b2-a179-5690da8d4752) Jest Tests #<span></span>19 / Paginated Table Component Events should update the page when the activePage is changed from redux
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d24-437c-a0fa-9f52f460647f) Jest Tests #<span></span>19 / Paginated Table Component rendering it renders the correct amount of pages and starts at activePage: 0
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddc-4fc2-ac34-8317421dd8ea) Jest Tests #<span></span>19 / Paginated Table Component rendering it renders the default load more table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-057a-4fa3-a6bb-f9c7d924e873) Jest Tests #<span></span>2 / PromptPage renders as expected with additional scripts
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-0578-4d0d-b0a4-ec487cd7f1c4) Jest Tests #<span></span>2 / PromptPage renders as expected without additional scripts
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee2-4bf5-bc1b-7def3f29fcfd) Jest Tests #<span></span>11 / query tab with unified timeline pagination should load more records according to sample size correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee2-4af0-b613-9f047b9a8f3b) Jest Tests #<span></span>11 / query tab with unified timeline pagination should load notes for correct page size
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee0-4676-883e-227ccd887ebd) Jest Tests #<span></span>11 / query tab with unified timeline pagination should load notes for current page only
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4245-8878-6dce9d6e4d5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4245-8878-6dce9d6e4d5d/artifacts/01965cef-fee0-43e1-9a85-15e4ba0095c4) Jest Tests #<span></span>11 / query tab with unified timeline pagination should paginate correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-0579-428e-8491-70239f62ebd5) Jest Tests #<span></span>2 / ResetSessionPage renders as expected
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-0579-4c86-9739-147418d60a67) Jest Tests #<span></span>2 / ResetSessionPage renders as expected with custom page title
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd01-4c31-9f18-b67e68b4f1da) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd01-4c31-9f18-b67e68b4f1da/artifacts/01965ced-206a-4079-a515-741f7af8eab4) Jest Tests #<span></span>8 / Response actions history page Read from URL params should read and set paging values from URL params
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddd-4f3e-b6a4-b5362f5e3fce) Jest Tests #<span></span>19 / Tls Table Component Rendering it renders the default Domains table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-0579-404a-84b2-49ab82cb4f08) Jest Tests #<span></span>2 / UnauthenticatedPage renders as expected
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd00-4703-a3a8-3798ba671260) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd00-4703-a3a8-3798ba671260/artifacts/01965cea-0578-4977-8233-6ee7564adb1d) Jest Tests #<span></span>2 / UnauthenticatedPage renders as expected with custom title
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8ddd-4821-867b-8a0aba7747ef) Jest Tests #<span></span>19 / Uncommon Process Table Component rendering it renders the default Uncommon process table
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294792#01965cc5-cd03-47be-8ca6-804e20e7a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294792/jobs/01965cc5-cd03-47be-8ca6-804e20e7a2c1/artifacts/01965ced-8d23-40b0-80b5-22757da7f163) Jest Tests #<span></span>19 / Users Table Component Rendering it renders the default Users table

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294792","number":294792,"commit":"7f7ea97b6c8f5bf82678f0929a583edca73e71c4"}],"number":294792}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294842)
* Commit: 93bd6478f1bfe750283bbe0f4ec76a6792916c0f

### Failed CI Steps
* [Jest Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366)
* [Jest Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d2c-54d2-40cb-848a-4c1319fca366/artifacts/01965d50-54af-436d-930d-e64e79dde034) Jest Tests #<span></span>3 / AlertsGrouping resets all levels pagination when global query updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d50-75fc-4306-bdb9-11f0fc687982/artifacts/01965d54-481c-4bc7-a6a4-52ecb5e1efa6) Jest Tests #<span></span>3 / AlertsGrouping resets all levels pagination when global query updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d2c-54d2-40cb-848a-4c1319fca366/artifacts/01965d50-54ae-4d45-a2b2-8a882a852fb2) Jest Tests #<span></span>3 / AlertsGrouping resets all levels pagination when selected group changes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d50-75fc-4306-bdb9-11f0fc687982/artifacts/01965d54-481d-4618-bc13-81fb2d2d33d8) Jest Tests #<span></span>3 / AlertsGrouping resets all levels pagination when selected group changes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d2c-54d2-40cb-848a-4c1319fca366/artifacts/01965d50-54b0-400c-8928-ea04a22f6428) Jest Tests #<span></span>3 / AlertsGrouping resets innermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d50-75fc-4306-bdb9-11f0fc687982/artifacts/01965d54-481d-470e-91af-a5359be51ba6) Jest Tests #<span></span>3 / AlertsGrouping resets innermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d2c-54d2-40cb-848a-4c1319fca366/artifacts/01965d50-54af-44a3-b2c6-fc18b6b26336) Jest Tests #<span></span>3 / AlertsGrouping resets only most inner group pagination when its parent groups open/close
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d50-75fc-4306-bdb9-11f0fc687982/artifacts/01965d54-481c-49cf-9c1f-105a189ad94d) Jest Tests #<span></span>3 / AlertsGrouping resets only most inner group pagination when its parent groups open/close
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d2c-54d2-40cb-848a-4c1319fca366) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d2c-54d2-40cb-848a-4c1319fca366/artifacts/01965d50-54ae-41be-a151-3d59a330a365) Jest Tests #<span></span>3 / AlertsGrouping resets outermost level's current page when that level's page size updates
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294842#01965d50-75fc-4306-bdb9-11f0fc687982) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294842/jobs/01965d50-75fc-4306-bdb9-11f0fc687982/artifacts/01965d54-481a-455f-9eea-18abe21a2036) Jest Tests #<span></span>3 / AlertsGrouping resets outermost level's current page when that level's page size updates

### History
* :broken_heart: [Build #294792](https://buildkite.com/elastic/kibana-pull-request/builds/294792) failed 7f7ea97b6c8f5bf82678f0929a583edca73e71c4

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294842","number":294842,"commit":"93bd6478f1bfe750283bbe0f4ec76a6792916c0f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294792","number":294792,"commit":"7f7ea97b6c8f5bf82678f0929a583edca73e71c4"}],"number":294842}
buildkite-pr-comment-->

mgadewoll committed:
test: update tests

- updates assertions on aria-current value from true to page, updated replaced icon

mgadewoll committed:
test: update snapshots

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294882)
* Commit: 55fa6df99e75f714a0b3285c787acb92541b5ccf

### Failed CI Steps
* [FTR Configs #20](https://buildkite.com/elastic/kibana-pull-request/builds/294882#01965d7e-93e8-43e4-98a7-9dd7fcd90155)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294882#01965d7e-93e8-43e4-98a7-9dd7fcd90155) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294882/jobs/01965d7e-93e8-43e4-98a7-9dd7fcd90155/artifacts/01965da3-1f4e-48eb-9ea6-1be469a449b2) FTR Configs #<span></span>20 / security APIs - Session Idle Session Idle cleanup should properly clean up session expired because of idle timeout when providers override global session config

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/af381920ee321835dfdfa01fd2d57f4f77ae3d4a) | [after](https://github.com/elastic/kibana/commit/55fa6df99e75f714a0b3285c787acb92541b5ccf) | diff |
| --- | --- | --- | --- |
| `controls` | 420.9KB | 420.9KB | +6.0B |
| `dashboard` | 551.4KB | 551.4KB | +6.0B |
| `maps` | 3.0MB | 3.0MB | +6.0B |
| `ml` | 5.3MB | 5.3MB | +6.0B |
| `visTypeVega` | 2.0MB | 2.0MB | +6.0B |
| total |  |  | +30.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/af381920ee321835dfdfa01fd2d57f4f77ae3d4a) | [after](https://github.com/elastic/kibana/commit/55fa6df99e75f714a0b3285c787acb92541b5ccf) | diff |
| --- | --- | --- | --- |
| `kbnUiSharedDeps-npmDll` | 6.2MB | 6.2MB | -39.0B |



### History
* :broken_heart: [Build #294842](https://buildkite.com/elastic/kibana-pull-request/builds/294842) failed 93bd6478f1bfe750283bbe0f4ec76a6792916c0f
* :broken_heart: [Build #294792](https://buildkite.com/elastic/kibana-pull-request/builds/294792) failed 7f7ea97b6c8f5bf82678f0929a583edca73e71c4

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294882","number":294882,"commit":"55fa6df99e75f714a0b3285c787acb92541b5ccf"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294842","number":294842,"commit":"93bd6478f1bfe750283bbe0f4ec76a6792916c0f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294792","number":294792,"commit":"7f7ea97b6c8f5bf82678f0929a583edca73e71c4"}],"number":294882}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/eui-team (EUI)

davismcphee reviewed (approved):
Data Discovery snapshot changes LGTM

tomsonpl reviewed (approved):
Defend Workflows LGTM

dmlemeshko reviewed (approved):
`src/platform/test/functional/services/data_grid.ts` changes LGTM

elena-shostak reviewed (approved):
LGTM!

machadoum reviewed (approved):
Entity analytics changes LGTM!

christineweng reviewed (approved):
Investigations changes LGTM

rylnd reviewed (approved):
Detection Engine changes (a single snapshot file) LGTM. I'll be fixing the overeager tests shortly, as well.

mgadewoll commented:
@elasticmachine merge upstream

elasticmachine committed:
Merge branch 'main' into eui/v101.4.0

markov00 reviewed (approved):
Owned code review, LGTM

kibanamachine commented:
Starting backport for target branches: 9.0

https://github.com/elastic/kibana/actions/runs/14643440801

kibanamachine commented:
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|9.0|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 218778
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

mgadewoll commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/219117">](https://github.com/elastic/kibana/pull/219117)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->
